fonts.css

```diff
@@ -1,27 +1,31 @@
 @font-face {
-  font-family: 'ptserif';
-  src: url('ptserif-bold-webfont.woff') format('woff');
+  font-family: 'PTSerif';
+  src: url('fonts/PTSerifBold.woff2') format('woff2'), 
```

Comment:
Codacy found an issue: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6240796)